google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/SelectConfigDestinationFolderDialog.java

```diff
@@ -53,6 +55,25 @@ public SelectConfigDestinationFolderDialog(@Nullable Project project) {
         project,
         FileChooserDescriptorFactory.createSingleFolderDescriptor()
     );
+
+    if (project != null && project.getBasePath() != null) {
+      String srcPath = project.getBasePath() + "/src";
+      String srcMainPath = project.getBasePath() + "/src/main";
+      String AppenginePath = project.getBasePath() + "/src/main/appengine";
```

Comment:
start variable name with a lower-case letter even if the word is normally capitalized (appEnginePath)